### PR TITLE
Search and filter both working

### LINKED ISSUE #54: Search/filter

### PR DESCRIPTION
**Pull request by SeanWallach**
I log all the search results in the Terminal so you can see why books are added (based on name, ISBN, inventory, etc)

The only issue is the search results ('library' attribute) are not displaying. They were earlier, then i added a bunch of stuff and something broke it. It should be a one/two line fix somewhere on userHomepage, related to the library variable

### CODE REVIEW

**Review by adybka — CHANGES_REQUESTED** [review]
Remove the compiler.xml file from the commit

File: src/main/resources/templates/userHomepage.html [hunk 1]
```diff
@@ -5,8 +5,14 @@
     <title>Amazin Main</title>
 </head>
 <body>
-<p>User Login Successful, Welcome to Book Store</p>
-<fieldset>
```
Comment (adybka) [inline src/main/resources/templates/userHomepage.html:9]: You removed this line `<fieldset>` so it seems there is a missing starting tag for the ending. fix this as well

**Comment by SeanWallach** [comment]
fixed and fixed

**Review by adybka — APPROVED** [review]

**Comment by adybka** [comment]
> fixed and fixed

All good from me then, we willl wait until @Ajfoster4444 Reviews it

**Comment by SeanWallach** [comment]
do you have any idea about the library attribute not being overwritten in userHomepage? after being added from the filter function in UserController

**Review by AjFroster — APPROVED** [review]

**Pull request merged**